core/src/test/resources/XmlFileElementIteratorPipe/input.xml

```diff
@@ -0,0 +1,426 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<GetPartiesOnAgreementRLY xmlns="http://www.ibissource.org/tom">
```

Comment:
Ik kan me voorstellen dat deze file wel wat ingedikt kan worden, zodat duidelijker is wat de structuur er van is, en wat de elementChain precies zou doen

Reply:
Beter zo, al moest ik wel behoorlijk goed kijken dat `Party` over gaat in `Partie`.